pkgs/servers/invidious/lsquic.nix

```diff
@@ -2,23 +2,23 @@
 let
   # lsquic requires a specific boringssl version (noted in its README)
   boringssl' = boringssl.overrideAttrs (old: rec {
-    version = "251b5169fd44345f455438312ec4e18ae07fd58c";
+    version = "a2278d4d2cabe73f6663e3299ea7808edfa306b9";
     src = fetchgit {
       url = "https://boringssl.googlesource.com/boringssl";
       rev = version;
-      sha256 = "sha256-EU6T9yQCdOLx98Io8o01rEsgxDFF/Xoy42LgPopD2/A=";
+      sha256 = "1m58rqnw8mp0219672qlw9iv16wz38rdfa29h4whxmrs1brqbc17";
     };
   });
 in
 stdenv.mkDerivation rec {
   pname = "lsquic";
-  version = "2.18.1";
+  version = "3.0.3";
```

Comment:
I didn’t test this yet, and I don’t know if I’ll have the time to test it in the next days, but AFAIK `lsquic.cr` always requires the lsquic version to match the tag of its version (see [README](https://github.com/iv-org/lsquic.cr#lsquic)). lsquic also requires a specific boringssl version, also specified in [its README](https://github.com/litespeedtech/lsquic#building-boringssl).

Reply:
You are probably right, so the update script should pick the lsquic version from *shard.lock*.